### PR TITLE
[SDK-882] Update examples to 1.0 APIs

### PR DESCRIPTION
**Pull request by jdm717**
## What

Updates the examples in this repository to use the 1.0 APIs.

The current state of the repo (with examples using the poc-viewer) is captured in [this branch](https://github.com/Vertexvis/web-sdk-examples/tree/poc-examples).

Note: This set of changes removes the `fit-to-part` example, as we do not currently receive information about the bounding box of a part when performing a pick.

## Ticket

https://vertexvis.atlassian.net/browse/SDK-882

@Vertexvis/sdk 
PTAL

### CODE REVIEW

File: examples/loading-models/index.html [hunk 1]
```diff
@@ -10,15 +10,15 @@
 
     <link
       rel="stylesheet"
-      href="https://unpkg.com/@vertexvis/viewer@0.3.2-8/dist/viewer/viewer.css"
+      href="https://unpkg.com/@vertexvis/viewer@0.9.3/dist/viewer/viewer.css"
```
Comment (brentav) [inline examples/loading-models/index.html]: @latest?

File: examples/picking/main.js [hunk 2]
```diff
@@ -1,28 +1,38 @@
...
+  await loadDefaultStreamKey(viewer);
 
   viewer.addEventListener('tap', async (event) => {
     const { position } = event.detail;
     const scene = await viewer.scene();
     const raycaster = await scene.raycaster();
 
-    const { bomItems } = await raycaster.intersectItems(position).execute();
+    const result = await raycaster.hitItems(position);
+    console.log(result);
```
Comment (danschultz) [inline examples/picking/main.js]: I'd say either remove this log, or add a prefix to it: `console.log('hit results', result)`

File: examples/work-instructions/instructions.js [hunk 3]
```diff
@@ -57,33 +36,48 @@ export async function applyCamera(scene, stepNumber) {
...
-  return operations.reduce((newScene, operation) => {
-    switch (operation.type) {
+function applyQuery(builder, query) {
+  switch (query.type) {
+    case 'all':
+      return builder.all();
+    case 'itemId':
+      console.log(
+        query.values.reduce((result, v) => result.withItemId(v).or(), builder)
+      );
```
Comment (danschultz) [inline examples/work-instructions/instructions.js]: same here

File: src/server.js [hunk 4]
```diff
@@ -1,26 +1,16 @@
...
 const app = express();
 app.use(cors());
 
-app.get(
-  '/token',
-  tokenHandler(oauth2Endpoint, clientId, clientSecret, expiresInSeconds)
-);
-
-app.get('/model', (_, res) => res.json({ urn: model }));
+app.get('/stream-key', (_, res) => res.json({ key: streamKey }));
```
Comment (danschultz) [inline src/server.js]: At this point, I'm not sure if we even need a Node backend. There's no communication back to Vertex.

File: examples/picking/main.js (hunk 2 above)
Comment (jdm717) [reply examples/picking/main.js]: Yeah, I think this was left over from some testing I was doing. I'll remove

File: examples/work-instructions/instructions.js (hunk 3 above)
Comment (jdm717) [reply examples/work-instructions/instructions.js]: Same with this log

File: src/server.js (hunk 4 above)
Comment (jdm717) [reply src/server.js]: Yeah, it currently isn't really necessary with these changes. I can remove it for now, and we can always add it back later if it ends up being needed

**Review by danschultz — APPROVED** [review]

**Pull request merged**